shipit.yml

```diff
@@ -1,5 +1,5 @@
 deploy:
   override:
-    - gem install bundler -v 1.17.3
+    - gem install bundler -v 1.3
```

Comment:
I'm not sure if this will work, but this version is explicitly required by `lita`.